variation/validators/duplication_deletion_base.py

```diff
@@ -0,0 +1,254 @@
+"""The base class for Duplication and Deletion Validation."""
+from typing import Optional, List, Dict, Tuple
+from variation.schemas.token_response_schema import Token
+from variation.validators.validator import Validator
+from variation.hgvs_dup_del_mode import HGVSDupDelMode
+from variation.data_sources import SeqRepoAccess, TranscriptMappings, UTA
+from variation.tokenizers import GeneSymbol
+from variation.mane_transcript import MANETranscript
+from ga4gh.vrs.dataproxy import SeqRepoDataProxy
+from ga4gh.vrs.extras.translator import Translator
+from gene.query import QueryHandler as GeneQueryHandler
+import logging
+from variation.schemas.normalize_response_schema\
+    import HGVSDupDelMode as HGVSDupDelModeEnum
+
+logger = logging.getLogger('variation')
+logger.setLevel(logging.DEBUG)
+
+
+class DuplicationDeletionBase(Validator):
```

Comment:
I wonder if it would be worth it to change up the class hierarchy a bit here? it seems like there are a bunch of abstract methods in Validator that this class doesn't need to implement

Reply:
Yes 😞 I originally wrote it so that these methods were in the Validator class, but moved it out to try and reduce how large that class is. Might just move it back there and refactor in #189

Reply:
Rather than moving back to validator class, I just raised a not implemented error